libsolidity/parsing/Token.cpp

```diff
@@ -160,6 +162,7 @@ tuple<Token::Value, unsigned int, unsigned int> Token::fromIdentifierOrKeyword(s
 					m % 8 == 0 &&
 					n % 8 == 0
 				) {
+					solAssert(n != -1, "Invalid type N in fixed command. Should not reach here.");
```

Comment:
This should work, but I think it would be clearer to just put `0 <= m &&m <= 256 && 0 <= n && n <= 256` into the `if`.